Frontend/src/Components/Dashboard/dataViewOptions.js

```diff
@@ -2,6 +2,7 @@
 const dvOptions = {
     battery_pack: "battery_pack",
     cell_groups: "cell_groups",
+    battery: "battery",

```

Comment:
```suggestion
    battery_pack: "battery_pack",
    cell_groups: "cell_groups",
```
Again, goes with the previous comments: Let's reuse the "battery_pack" option rather than creating a new one

Reply:
Unresolved. Please remove the "battery" option

Reply:
My bad, good catch